core/domain/user_jobs_continuous.py

```diff
@@ -449,13 +450,14 @@ def get_dashboard_stats(cls, user_id):
         Args:
             user_id: str. The ID of the user.
 
-        Returns a dict with the following keys:
-            'total_plays': int. Number of times the user's explorations were
-                played.
-            'num_ratings': int. Number of times the explorations have been
-                rated.
-            'average_ratings': float. Average of average ratings across all
-                explorations.
+        Returns:
+            dict. with the following keys:
```

Comment:
dict. This has the following keys: ...

(Use complete sentences please.)

Reply:
Done.